src/clr-core/styles/tokens/tokens.stories.mdx

```diff
@@ -0,0 +1,85 @@
+import { Meta, Story, Preview, API } from '@storybook/addon-docs/blocks';
+import { html, LitElement } from 'lit-element';
+
+<Meta title="Experimental/Design Tokens/Getting Started" />
+
+# Design Tokens
+
+<cds-alert status="danger" closable="false" cds-layout="m-b:md">
+  <cds-alert-content>This is a experimental API and will change. Do not use this in production enviroments!</cds-alert-content>
+</cds-alert>
+
+Design Tokens are global variables to configure the foundation of the
+design system. Changes to a design token will propagate throughout the entire system.
+Tokens are broken down into five categories:
+
+- [Spacing](./?path=/docs/experimental-spacing-getting-started--page)
+- [Layout](./?path=/docs/experimental-layout-get-started--page)
+- [Typography](./?path=/docs/experimental-typography-getting-started--page)
+- [Color](./?path=/docs/experimental-color-getting-started--page)
+- Global
+
+## Installation
+
+To use the design token system via CSS import the global CSS file.
+
+```scss
+// import into sass or css
+@import '~@clr/core/styles/module.tokens.min';
+
+// or import in html
+<link href="/node_modules/@clr/core/styles/module.tokens.min.css" rel="stylesheet">
+```
+
+## Usage
+
+Tokens can be adjusted by setting the values on the `:root` selector.
+
+```css
+:root {
+  --cds-token-typography-body-color: rgb(26, 26, 26);
+}
+```
+
+## Base Font
+
+By default Clarity has a base font size of 20px on the document. All CSS px values
+relative to this base font. If your application is already using a base 16px font
+size you can set the following:
+
+```html
+<html lang="en" cds-base-font="16">
+```
+
+By adding the `cds-base-font="16"` to the root html tag this will adjust all spacing
```

Comment:
`Adding cds-base-font="16" to the root html tag will adjust all spacing and px values in Clarity` not `By adding the `cds-base-font="16"` to the root html tag this will adjust all spacing
and px values in Clarity`